src/quiz/Description.jsx

```diff
@@ -69,12 +69,12 @@ export default function Description() {
     }, 1000);
   }, []);
 
-  if (time == 0) history.push("/answer");
+  if (time === 0) history.push("/answer");
 
   return (
     <div id="description">
       <div className="title">この画像を説明しよう</div>
-      <img src="https://source.unsplash.com/featured/?lion" />
+      <img src="https://source.unsplash.com/featured/?lion" alt="" />
```

Comment:
```suggestion
      <img src="https://source.unsplash.com/featured/?lion" alt="問題の画像" />
```

Altつけるなら空白じゃなくてちゃんとそこが何を表しているかをつけましょう